restore/action.yml

```diff
@@ -0,0 +1,27 @@
+name: 'Restore Cache'
+description: 'Restore Cache artifacts like dependencies and build outputs to improve workflow execution time'
+author: 'GitHub'
+inputs:
+  path:
+    description: 'A list of files, directories, and wildcard patterns to cache and restore'
```

Comment:
This is only for restore now.

Reply:
Forgot to reword it, I was thinking of something like `The list of files, directories, and wildcard patterns to restore cache that were used while saving it`

Reply:
Updated